joint_trajectory_controller/src/joint_trajectory_controller_parameters.yaml

```diff
@@ -0,0 +1,73 @@
+joint_trajectory_controller:
+  joints: {
+    type: string_array,
+    default_value: [],
+    description: "Names of joints used by the controller",
```

Comment:
So is this going to replace the list of interfaces in the URDF? I'm not sure about that.

It's nice having it in the URDF so it applies to all controllers, not just the JTC. It looks like what you have here will need to be duplicated a lot.

Reply:
No, this just replaces the YAML parameter declaration and getting. This does not change anything about how interfaces from the URDF work.